workgraph/windows.yml

```diff
@@ -82,39 +82,46 @@ windows-fix-desktop-leaks:
         greening tests, as the workers will just report "blue" jobs and fail,
         but needs to be fixed for tier 2.
 
-windows-fix-appdata:
-    title: "Fix issues with APPDATA being set incorrectly"
-    bug: 1319453
-    assigned: pmoore
+windows-sign-update-exe:
+    title: "sign test tools in windows build process"
+    bug: 1323732
+    assigned: grenade
+    description: |
+	    xpcshell and browser-chrome have tests which depend on update.exe being
+		available and signed.  We can build with |MOZ_UPDATER=| and avoid these
+		tests, or sign the test binaries which required signing infrastructure.
+
+windows-bbb-test-jobs:
```

Comment:
We already have `windows-hardware-via-bbb-green` for this.  We can't actually run the test jobs via BBB in tier 2, because we don't have enough capacity to run double the load on hardware.  So we will make sure those BBB jobs run by testing in try.  There's no worry about some weird test case failing via BBB since the tests are not actually running in a new environment -- we just need to make sure that the BBB successfully triggers them.